docs/APIReviewProcess.md

```diff
@@ -24,7 +26,7 @@ Before marking an issue as `api-ready-for-review`, make sure that the issue has
 * A short description that will help reviewers not familiar with this area.
 * The API changes in ref-assembly format. It's fine to link this to the generated ref-assembly-code in the PR. If the changes are to an area that does produce ref-assemblies, please write out what it would look like in ref-assembly format for us to review.
 
-```txt
+````text
```

Comment:
four ticks? Surprised it's happy about that 😄

Reply:
I see this in a few other files too.

Reply:
how curious

Reply:
turns out that it's intentional
https://github.github.com/gfm/#fenced-code-blocks
similar to https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#raw-string-literals
it's so you can include stuff that itself has 3 ticks.

I'll fix though.

Reply:
hmm, but it should have detected they weren't balanced...

Reply:
or more strictly, that they weren't balanced by the end of the file.

fixed

Reply:
Yeah I change between one tick and three ticks all the time when doing comments on issues/PRs. So its definitely a standard markdown thing.